stickynotes/stickynotes.c

```diff
@@ -823,14 +821,16 @@ stickynotes_save_now (void)
 
     xmlDocSetRootElement (doc, root);
     xmlNewProp (root, XML_CHAR ("version"), XML_CHAR (VERSION));
-
-    wnck_screen = wnck_screen_get_default ();
-    wnck_screen_force_update (wnck_screen);
-
+#ifdef GDK_WINDOWING_X11
+    GdkDisplay *display = gdk_screen_get_display(gdk_screen_get_default());
+    if (GDK_IS_X11_DISPLAY(display))
```

Comment:
```suggestion
    GdkDisplay *display = gdk_screen_get_display (gdk_screen_get_default());
    if (GDK_IS_X11_DISPLAY (display))
```

Reply:
Easy to handle, can do it now

Reply:
Got that one too, it's in squashed-to-one